Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitWebXR.cpp

```diff
@@ -88,7 +98,28 @@ class WebXRTest : public WebViewTest {
             waitUntilIsImmersiveModeEnabledChanged();
     }
 
+    GRefPtr<WebKitSettings> customSettings() const override
+    {
+        GRefPtr<WebKitSettings> webkitSettings = adoptGRef(webkit_settings_new());
+        g_autoptr(WebKitFeatureList) featureList = webkit_settings_get_development_features();
+        WebKitFeature* feature = findFeature(featureList, "OpenXRDMABufRelaxedForTesting");
+        g_assert_nonnull(feature);
+        webkit_settings_set_feature_enabled(webkitSettings.get(), feature, true);
```

Comment:
Isn't it possible to change the setting instead of having to create the view with the setting already set?

Reply:
I prefer if it's changed after instead of adding a new virtual to WebView class that is not clear what it's for.

Reply:
done